src/format-message.ts

```diff
@@ -49,7 +49,8 @@ const getSummary = (
       checkSchemaResult?.diffToPrevious.severity === 'WARNING' ||
       checkSchemaResult?.diffToPrevious.severity === 'FAILURE'
     ) {
-      summary += `❌ Found **${checkSchemaResult?.diffToPrevious.changes.length} breaking changes**\n\n`;
+      const issueCount = checkSchemaResult?.diffToPrevious.changes.filter((c) => c.severity === 'FAILURE' || c.severity === 'WARNING').length;
```

Comment:
I'm not sure if warnings should be included in this count either. Curious what you think.

```suggestion
      const issueCount = checkSchemaResult?.diffToPrevious.changes.filter((change) => change.severity === 'FAILURE' || change.severity === 'WARNING').length;
```

Reply:
I think because both cases cause the overall failure, they both should be included in the count. I've been away from it for a few days, but iirc, I remember the same surprise as you. I think I confirmed that warnings are actually breaking changes too and so should be flagged